js/api-augment/definitions/schemas.ts

```diff
@@ -1,61 +1,71 @@
 export default {
   rpc: {
-    getBySchemaId: {
```

Comment:
is this one of those RPC's that Peter is using? If so might need to keep it

Reply:
Yes; restored in a later commit.